app/models/source.rb

```diff
@@ -0,0 +1,8 @@
+class Source < ActiveRecord::Base
+  belongs_to :source_set
+  validates :aggregation, presence: true
+
+  def aggregation_uri
+    'http://dp.la/item/' + aggregation
```

Comment:
Can we make this URI base configurable? This will need to be different for staging vs. production.